README.md

```diff
@@ -1,12 +1,12 @@
 # Karmatic [![npm](https://img.shields.io/npm/v/karmatic.svg)](https://npm.im/karmatic) [![travis](https://travis-ci.org/developit/karmatic.svg?branch=master)](https://travis-ci.org/developit/karmatic)
 
-A simplified zero-configuration wrapper around [Karma], [Webpack], [Jasmine] & [Puppeteer].
+A simplified zero-configuration wrapper around [Karma], [Rollup] or [Webpack], [Jasmine] & [Puppeteer].
```

Comment:
Any preferences on how to talk about Rollup here?

Reply:
```suggestion
Zero-config browser tests powered by [Karma] & [Puppeteer], with automatic [Rollup]/[Webpack] support.
```